acceptance/tests/concurrency/ticket_2659_concurrent_catalog_requests.rb

```diff
@@ -87,7 +87,7 @@
           sleep $sleep_for
           url='https://#{master}:8140/puppet/v3/catalog/#{agent_cert}?environment=production'
           echo "Curling: $url"
-          env PATH='#{agent[:privatebindir]}:${PATH}' curl --tlsv1 -v -# -H 'Accept: text/pson' --cert #{cert_path} --key #{key_path} --cacert #{cacert_path} $url
+          env PATH="#{agent[:privatebindir]}:$PATH" curl --tlsv1 -v -# -H 'Accept: text/pson' --cert #{cert_path} --key #{key_path} --cacert #{cacert_path} $url
```

Comment:
So I think using double quotes is the correct thing to do.

I think what happened is we used single quotes resulting in `env PATH='/opt/puppetlabs/puppet/bin/:$PATH' curl ...` which corrupts the PATH when trying to execute curl. However, it has worked because we build and distribute curl in puppet-agent, and we want to use ours, so we're sure it supports tlsv1 and isn't some archiac system openssl.

The part that's strange is why does the current test fail on huaweios? Are we not building/including curl on that platform? Does it rely on other executables which are not in puppet's privatebindir?